### PR TITLE
fix discord dead link (never expire & no limit)

### PR DESCRIPTION
**Pull request by felixbbertrand**

### CODE REVIEW

**Comment by felixbbertrand** [comment]
@luxumbra 👋

**Comment by felixbbertrand** [comment]
a quick fix @peth-yursick

File: docs/enter-metagame/join-metagame.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Eager to support the cause & impatient to join?
 You can skip the queue by joining as a Patron instead!
 - Pick up a minimum of 5 Seeds [here](https://balancer.exchange/#/swap/ether/0x30cf203b48edaa42c3b4918e955fed26cd012a3f)
 - Plant them in [here](https://pools.balancer.exchange/#/pool/0xea05a15dbce2eb543ffda16950e95b2bd2e40d0e/)
-- Enter [here](https://discord.gg/Hf54gd8) & type in "!join"
+- Enter [here](https://discord.gg/d3rurFAK6M) & type in "!join"
```
Comment (luxumbra) [inline docs/enter-metagame/join-metagame.md:21]: Sweet.

**Review by luxumbra — APPROVED** [review]
Thanks @felixbbertrand

**Comment by luxumbra** [comment]
@felixbbertrand do you want to merge this or do you not have permissions?

**Comment by felixbbertrand** [comment]
@luxumbra dont have any permission

**Pull request merged**